packages/cli/src/ui/App.tsx

```diff
@@ -420,15 +421,16 @@ const App = ({ config, settings, startupWarnings = [], version }: AppProps) => {
 
   // Terminal and UI setup
   const { rows: terminalHeight, columns: terminalWidth } = useTerminalSize();
+  const isNarrow = terminalWidth < NARROW_BREAKPOINT;
```

Comment:
see this a lot of places. I would add it as a helper function as its intent is clearer than the name NARROW_BREAKPOINT.